spec/rails_app/app/models/dummy.rb

```diff
@@ -1,3 +1,4 @@
 class Dummy < ApplicationRecord
-  validates :name, :number, presence: true
+  validates :name, presence: true
+  validates :number, presence: true # uniqueness: true
```

Comment:
Same question here about the comment

Reply:
In this case, `:number` has the unique constraint in the database, but if I use this validation in this dummy model, it's going to raise an `ActiveModel::ValidationError` instead of ActiveRecord::RecordNotUnique